README.md

```diff
@@ -61,6 +67,77 @@ yarn start
 
 Finally, we can navigate a browser window to `localhost:8080` to view generated data.
 
+### Running with Docker
+
+Optionally, you can build and run sourcecred in a container to avoid installing dependencies
+on your host. First, build the container:
+
+```bash
+$ docker build -t sourcecred .
+```
+
+If you don't want to build, the container is also provided at [vanessa/sourcecred](https://hub.docker.com/r/vanessa/sourcecred)
+
+```bash
+$ docker pull vanessa/sourcecred
+$ docker tag vanessa/sourcecred sourcecred
+```
+
+You will still need to export a GitHub token, and then provide it to the container
+when you run it. Notice that we are also binding port 8080 so we can
+view the web interface that will be opened up.  The only argument needed is the 
+GitHub repository to generate the sourcecred for:
+
+```bash
+REPOSITORY=sfosc/sfosc
+$ docker run -d --name sourcecred --rm --env SOURCECRED_GITHUB_TOKEN=$(cat .token) -p 8080:8080 sourcecred $REPOSITORY
```

Comment:
This is also unsafe as written because `"$(cat .token)"` needs to be
quoted, lest it be subject to glob expansion and word splitting. So does
`"${REPOSITORY}"`, and you should probably include a `--` before that
argument to prevent the repository name from being interpreted as a
Docker option.

Reply:
yep we discussed this in your first comment! It won't be interpreted as a docker option because it's coming after the container name.

Reply:
> It won't be interpreted as a docker option because it's coming after
> the container name.

Good to know; thanks.